server/users.ts

```diff
@@ -186,24 +169,25 @@ function findUsers(userids, ips, options = {}) {
  * User groups
  *********************************************************/
 
-let usergroups = Object.create(null);
+const usergroups = Object.create(null);
 function importUsergroups() {
 	// can't just say usergroups = {} because it's exported
-	for (let i in usergroups) delete usergroups[i];
-
+	for (const i in usergroups) delete usergroups[i];
+	// tslint:disable-next-line:no-floating-promises
```

Comment:
This should actually be `return FS(....)`, and then at the callsite on lint 193 is where we should be ignoring